lib/modules/dosomething/dosomething_reportback/includes/Reportback.php

```diff
@@ -92,6 +92,14 @@ public static function get($ids) {
 
     $results = dosomething_reportback_get_reportbacks_query(['rbid' => $ids]);
 
+    if (!user_access('administer modules') && !$results) {
+      $results = dosomething_reportback_get_reportbacks_query(['rbid' => $ids], TRUE);
```

Comment:
I think needing to do the same call but with an added param here just feels like a bad pattern and what's leading to confusion. I feel there's likely a better way of handling this, just not sure what it is offhand... maybe we could chat it out in Slack?